### PR TITLE
DIT-514: fixing mongo TTL

### PR DESCRIPTION
**Pull request by gokyo**

### CODE REVIEW

File: app/uk/gov/hmrc/bindingtarifffilestore/connector/AmazonS3Connector.scala [hunk 1]
```diff
@@ -70,16 +71,18 @@ class AmazonS3Connector @Inject()(config: AppConfig) {
...
 
-    s3client.putObject(request)
+    Try ( s3client.putObject(request) ) match {
+      case Success(_) =>
+        val authenticatedURLRequest = new GeneratePresignedUrlRequest(config.s3Configuration.bucket, fileMetaData.id)
+          .withMethod(HttpMethod.GET)
+        val authenticatedURL: URL = s3client.generatePresignedUrl(authenticatedURLRequest)
+        fileMetaData.copy(url = Some(authenticatedURL.toString))
+      case Failure(e: Throwable) => throw e
+    }
```
Comment (ed0906) [inline app/uk/gov/hmrc/bindingtarifffilestore/connector/AmazonS3Connector.scala]: What's the benefit of this change if it just rethrows the exception? That's what the code did before?

File: conf/application.conf [hunk 2]
```diff
@@ -136,7 +136,7 @@ auditing {
 
 mongodb {
   uri = "mongodb://localhost:27017/"${appName}
-  timeToLiveInSeconds = 86400 # 24 hours   # TODO: we need 604800 (7 days) in MDTP
+  timeToLiveInSeconds = 3600
```
Comment (ed0906) [inline conf/application.conf]: is an hour long enough?

Comment (ed0906) [reply conf/application.conf]: ignore, It's overridden in other environments

File: app/uk/gov/hmrc/bindingtarifffilestore/connector/AmazonS3Connector.scala (hunk 1 above)
Comment (gokyo) [reply app/uk/gov/hmrc/bindingtarifffilestore/connector/AmazonS3Connector.scala]: I can remove the `Failure` case, but there will be a warning.

The code before was not linking the `putObject` instruction together with the rest of the code

**Pull request merged**